### ISSUE TITLE
Load paas-templates profile script in user profiles

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/orange-cf-bosh-cli#60]
As a paas-templates operator, I need to invoke alias/functions defined in paas-templates to workaround some limitations in CF CLI V7 beta

See https://github.com/orange-cloudfoundry/osb-cmdb-spike/issues/10#issuecomment-567967267

```bash
$ function cf_labels_service () { local SERVICE_NAME=$1; local SERVICE_GUID=$(cf service ${SERVICE_NAME} --guid); cf curl "/v3/service_instances?label_selector=backing_service_instance_guid==${SERVICE_GUID}" | jq .resources[].metadata ;}

$ cf s
Getting services in org osb-cmdb-backend-services-org-client-0 / space p-mysql as xx...

name                                           service   plan   bound apps   last operation     broker    upgrade available
p-mysql-80134f9b-b6fd-48e2-8ca5-e185c4cb5ce0   p-mysql   10mb                create succeeded   p-mysql   

$ cf_labels_service p-mysql-80134f9b-b6fd-48e2-8ca5-e185c4cb5ce0
{
  "labels": {
    "backing_service_instance_guid": "a29a0e9e-62be-409d-85cf-084daed9b619",
    "brokered_service_originating_identity_user_id": "683ea748-3092-4ff4-b656-39cacc4d5360",
    "brokered_service_instance_guid": "80134f9b-b6fd-48e2-8ca5-e185c4cb5ce0"
  },
  "annotations": {
    "brokered_service_context_organization_name": "system_domain",
    "brokered_service_context_spaceGuid": "568e6244-471b-466c-a372-3ab2109b69aa",
    "brokered_service_context_space_name": "osb-cmdb-broker",
    "brokered_service_context_instance_name": "test-mysql-cmdb-2",
    "brokered_service_context_organizationGuid": "87505508-4c09-499e-933c-eb89582cb9a5"
  }
}
```

As a paas-templates author, in order to update such scripts without creating a new orange-cf-bosh-cli docker bosh release, I need a script in paas-templates such as `ops-depls/cf-apps-deployments/osb-cmdb-broker/bosh-cli-profile.bash` to be invoked when user profile start.

**Comment by gberche-orange** [comment]
@ogrand 
the script to source in docker bosh-cli is available at https://github.com/orange-cloudfoundry/cf-cli-cmdb-scripts.git to be included as a submodule and added to docker bosh cli

**Comment by ogrand** [comment]
Closed in 2.1.38 release
